textproc/vislcg3/Portfile

```diff
@@ -25,14 +29,14 @@ long_description        Constraint Grammar (CG) is a methodological paradigm \
                         rule may be conditioned upon each other, negated or \
                         blocked by interfering words or tags.
 
-homepage                http://beta.visl.sdu.dk/constraint_grammar.html
-master_sites            http://beta.visl.sdu.dk/download/vislcg3/
+homepage                https://visl.sdu.dk/constraint_grammar.html
 
-checksums               md5    3875d7d36bedebe17317e32442c18b80 \
-                        sha1   8718fee3e8e9bb2edd58c11175b23661539309c1 \
-                        rmd160 fb0576a1f142bd47dfd35fb665408fe5dc9058e4
+checksums               rmd160 393f7cb49d40c265a1fcd29c13b999278712ee8d \
+                        sha256 ada9dd9f7cdd47d050f53fb87f5f4d590f899b7e5629e9fff694397a6b2bcd05 \
+                        size   357195
 
-depends_build           port:pkgconfig port:autoconf port:automake
```

Comment:
Since you are now using portgroups, this command overrides the dependencies from them. Thus you needed to manually add cmake as a build dependency. To fix that, use `depends_build-append` instead. Then you don't need to add cmake as a build dependency here.